tests/Type/Doctrine/Query/MysqlQueryResultTypeWalkerTest.php

```diff
@@ -0,0 +1,17 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Type\Doctrine\Query;
+
+/**
+ * @group platform
+ * @group mysql
+ */
+class MysqlQueryResultTypeWalkerTest extends PDOQueryResultTypeWalkerTest
```

Comment:
Currently I extends PDOQueryResultTypeWalkerTest but later we'll work his own dataprovider